features/org.jkiss.dbeaver.db.feature/feature.xml

```diff
@@ -88,6 +88,12 @@
          install-size="0"
          version="0.0.0"
          unpack="false"/>
+   <plugin
+           id="org.jkiss.dbeaver.ext.databend.ui"
```

Comment:
```suggestion
           id="org.jkiss.dbeaver.ext.databend"
```

Reply:
You registered your UI plugin part in the org.jkiss.dbeaver.db.ui.feature/feature.xml